testing/web3signer_tests/src/lib.rs

```diff
@@ -901,13 +901,14 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn prater_base_types() {
-        test_base_types("prater", 4246).await
+    async fn mainnet_bellatrix_types() {
+        test_base_types("mainnet", 4244).await
```

Comment:
Looks like we're testing base types here
```suggestion
        test_bellatrix_types("mainnet", 4244).await
```

Reply:
good catch!